tensorflow/lite/micro/examples/micro_speech/CMSIS/Makefile.inc

```diff
@@ -18,7 +18,6 @@ ifneq ($(filter CMSIS,$(ALL_TAGS)),)
     tensorflow/lite/micro/examples/micro_speech/CMSIS/hanning.h \
     tensorflow/lite/micro/examples/micro_speech/CMSIS/sin_1k.h \
     third_party/CMSIS_ext/README.md \
```

Comment:
As mentioned in #35889 I also had to remove this line in order for the project to build properly.